test/common/include/alpaka/test/mem/view/Iterator.hpp

```diff
@@ -0,0 +1,201 @@
+/**
+* \file
+* Copyright 2014-2016 Erik Zenker
+*
+* This file is part of alpaka.
+*
+* alpaka is free software: you can redistribute it and/or modify
+* it under the terms of the GNU Lesser General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* alpaka is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU Lesser General Public License for more details.
+*
+* You should have received a copy of the GNU Lesser General Public License
+* along with alpaka.
+* If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include <alpaka/alpaka.hpp>
+
+namespace alpaka
+{
+    //-----------------------------------------------------------------------------
+    //! The test specifics.
+    //-----------------------------------------------------------------------------
+    namespace test
+    {
+        //-----------------------------------------------------------------------------
+        //! The test mem specifics.
+        //-----------------------------------------------------------------------------
+        namespace mem
+        {
+            //-----------------------------------------------------------------------------
+            //! 
+            //-----------------------------------------------------------------------------
+            namespace view
+            {
+                //-----------------------------------------------------------------------------
+                //! 
+                //-----------------------------------------------------------------------------
+                namespace traits {
+
+                    //#############################################################################
+                    //! 
+                    //#############################################################################
+                    template<
+                        typename TView,
+                        typename TSfinae = void>
+
+                    struct IteratorType
+                    {
+                        using Dim  = alpaka::dim::Dim<TView>;
+                        using Size = alpaka::size::Size<TView>;
+                        using Elem = alpaka::elem::Elem<TView>;
+
+                        IteratorType(TView & view) :
```

Comment:
Could you please move each parameter indented by 4 spaces on an extra line in all functions?